Sources/HTTPTypes/HTTPField.swift

```diff
@@ -181,8 +195,43 @@ extension HTTPField: CustomPlaygroundDisplayConvertible {
     }
 }
 
+extension HTTPField: Codable {
+    enum CodingKeys: String, CodingKey {
+        case name
+        case value
+        case indexingStrategy
+    }
+
+    public func encode(to encoder: Encoder) throws {
+        var container = encoder.container(keyedBy: CodingKeys.self)
+        try container.encode(self.name, forKey: .name)
+        try container.encode(self.rawValue._storage, forKey: .value)
+        if self.indexingStrategy != .automatic {
+            try container.encode(self.indexingStrategy.rawValue, forKey: .indexingStrategy)
+        }
+    }
+
+    private enum DecodingError: Error {
+        case invalidValue(String)
+    }
```

Comment:
Can't we use one of the `DecodingError`s that are in the stdlib here?

Reply:
Fixed in e885fa315c7eba05c02f35aed7a8ea8dbe563b5e